packages/app-mobile/components/NoteBodyViewer/hooks/useEditPopup.test.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { editPopupClass, getEditPopupSource } from './useEditPopup';
+import { describe, it, expect, jest } from '@jest/globals';
+
+const createEditPopup = (target: HTMLElement) => {
+	const { createEditPopupSyntax } = getEditPopupSource();
+	eval(`(${createEditPopupSyntax})`)(target, 'someresourceid', '() => {}');
+};
+
+const destroyEditPopup = () => {
+	const { destroyEditPopupSyntax } = getEditPopupSource();
```

Comment:
This needs to be refactored — I don't like exporting both `getEditPopupSource` and `useEditPopup` from `useEditPopup.ts`. Perhaps the code for the edit popup can live in `packages/renderer`?

Reply:
At least for now, `useEditPopup` makes sense in `packages/app-mobile`:
1. `packages/renderer` currently doesn't seem to be linked to `@joplin/lib`, thus localizing strings with `_(...)` will be difficult.
2. `useEditPopup` returns CSS in addition to JavaScript strings. With `useEditPopup` structured as it is currently, it's easy to inject the CSS into the WebView. Moving calls to `useEditPopup` to the renderer might make this more difficult.